internal/store/ingress.go

```diff
@@ -141,11 +141,13 @@ func ingressMetricFamilies(allowAnnotationsList, allowLabelsList []string) []gen
 				for _, rule := range i.Spec.Rules {
 					if rule.HTTP != nil {
 						for _, path := range rule.HTTP.Paths {
-							ms = append(ms, &metric.Metric{
-								LabelKeys:   []string{"host", "path", "service_name", "service_port"},
-								LabelValues: []string{rule.Host, path.Path, path.Backend.Service.Name, strconv.Itoa(int(path.Backend.Service.Port.Number))},
-								Value:       1,
-							})
+							if path.Backend.Service != nil {
```

Comment:
Thanks for this PR. I wonder if we can just set `service_name` and `service_port` to blank values instead of not adding exporting a metric at all.

Reply:
Thanks for your comment! 🙇‍♂️ 

I updated it to return blank values("\<none\>") refer to other code.
Please tell me if I don't understand correctly.

And I added test code to the `ingress_test.go` file.